.github/workflows/ci_test.yml

```
@@ -92,6 +92,12 @@ jobs:
           cd ${{ github.workspace }}\oxen-rust
           mkdir .\data\test\runs
 
+      - name: Set up AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.9)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)